### DISCUSSION TITLE
文件管理器 切换tab快捷键

### DISCUSSION THREAD

**Discussion by HUSTERGS** [linuxdeepin/developer-center#2967] [Features Request & Ideas | 特性请求 & 头脑风暴]
希望可以添加`Ctrl+PageUp/PageDown`作为不同`tab`之间切换的快捷键(虽然`Alt+Num`可以选定特定`tab`，但是还是感觉`Ctrl+PageUp/PageDown`更符合习惯，毕竟浏览器和一些其他的文件管理器都是如此)

**Comment by meiyixiang** [comment]
你好关于无鼠标情况下的快捷键切换操作的支持，我们已经在解决中。
